### ISSUE TITLE
The "open" button should be a link

### ISSUE THREAD

**Issue by KewlCat** [SSilence/selfoss#1019]
The "open" button should be a link, since its functionality is to open the element ("article", ...) in the browser. But being a button element, there is no indication as to its destination, and there is no choice at all on its behaviour. For instance, I'd like to open an article in a new tab for later reading but not leave the list => I'd middle click on "open" (or Ctrl-click, or right-click + open in a new tab, ...), have it open in the background for later, and continue browsing the other articles.

**Comment by aslmx** [comment]
I agree. To help you out quickly: the timestamp is a link you can middle click to Open it in a new Tab.

**Comment by KewlCat** [comment]
I'll use the timestamp, then. Thanks !

**Comment by niol** [comment]
This is a duplicate of #963 .
